news/_posts/core-weekly/2020-10-19-coreweekly-42-2020.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 42 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-10-20
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 12th to Sunday 18th of October 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Last week a major event happened in PrestaShop history: **the maintainer team welcomed a new member that is not a PrestaShop employee!** Long-time contributor [@kpodemski](https://github.com/kpodemski) was granted the maintainer status after [he submitted his application](https://github.com/PrestaShop/open-source/issues/44) on the 17th of September. This is a very important milestone for the project.
+
+Following this amazing news, if you speak French you might be interested into the [PrestaShop Day France Online](https://events.prestashop.com/prestashop-day-paris-2020/) that happens today Tuesday 20th of October from 9:15am (Paris time) to 5:45pm. Live sessions are scheduled for the whole day and will cover multiple topics, speakers being PrestaShop employees or from the PrestaShop community, merchants and developers.
+
+In the meantime, all the `major` and `critical` bugs reported in PrestaShop 1.7.7.0 Release Candidate 1 first build have been fixed in the `1.7.7.x` branch. A new build has been delivered to QA team for, once again, testing before public release. Well, the testing will probably start tomorrow as QA team will be busy with the PrestaShop Day event.
+
+If you can't wait to test PrestaShop 1.7.7.0 Release Candidate, you can warm up by testing the [php8.0.0 Release Candidate 2](https://www.php.net/archive/2020.php#2020-10-16-1) published on the 16th of October 2020.
+
+
+## Releases
+
+* [dashtrends](https://github.com/PrestaShop/dashtrends) module: [v2.0.3](https://github.com/PrestaShop/dashtrends/releases/tag/v2.0.3)
+* [welcome](https://github.com/PrestaShop/welcome) module: [v6.0.2](https://github.com/PrestaShop/welcome/releases/tag/v6.0.2)
+* [ps_checkpayment](https://github.com/PrestaShop/ps_checkpayment) module: [v2.0.5](https://github.com/PrestaShop/ps_checkpayment/releases/tag/v2.0.5)
```

Comment:
```suggestion
* [ps_checkpayment](https://github.com/PrestaShop/ps_checkpayment) module: [v2.0.5](https://github.com/PrestaShop/ps_checkpayment/releases/tag/v2.0.5)
* [welcome](https://github.com/PrestaShop/welcome) module: [v6.0.2](https://github.com/PrestaShop/welcome/releases/tag/v6.0.2)
```

Reply:
I dont see the logic but if that makes you happy 😅

Reply:
The logic is the alphabetic order :rofl:

Reply:
I retrieve this list from GitHub API 🤔  so I guess there must be an order (not the alphabetical). Maybe by release date ?